### PR TITLE
update to new download apis; better docs

### PR DESCRIPTION
**Pull request by rosibaj**

### CODE REVIEW

File: docs/submission/submitting-molecular-data.md [hunk 1]
```diff
@@ -134,9 +134,36 @@ If the file(s) successfully upload, then you will receive an `Upload completed`
...
+
+
+### Troubleshooting help
+
+
+#### Upload error
+
+During upload, a temporary file is written to the directory where the file that is being uploaded is located.  If you do not have permission to write to this directory, the upload will fail.  To address this, update the score-client `conf/application.properties` configuration file with a `client.uploadStateDir` parameter.
+
+```yaml title="score-client config"
```
Comment (joneubank) [inline docs/submission/submitting-molecular-data.md:152]: need to put the first line of content beneath the ``` line or it wont get shown.

**Review by joneubank — CHANGES_REQUESTED** [review]
Found one mistake in a code block. The rest is all great. "better docs" is the feature we've all been waiting for ;)

File: docs/submission/submitting-molecular-data.md (hunk 1 above)
Comment (rosibaj) [reply docs/submission/submitting-molecular-data.md:152]: i dont think so - this isnt content.  its formatting and the title of the block

Comment (joneubank) [reply docs/submission/submitting-molecular-data.md:152]: yo this is a title to the code block. i've actually never seen this before, very cool.

**Review by joneubank — APPROVED** [review]

**Review by joneubank — APPROVED** [review]

**Pull request merged**